src/inspect_ai/model/_providers/openai.py

```diff
@@ -151,7 +151,7 @@ def __init__(
                 )
 
         # create async http client
-        http_client = OpenAIAsyncHttpxClient()
+        http_client = model_args.get('http_client', OpenAIAsyncHttpxClient())
```

Comment:
I think we want this to be `.pop()` so the arg is removed (otherwise we could get duplicate arguments in the `AsyncAzureOpenAI` constructor)